src/hooks-worker-client.coffee

```diff
@@ -301,7 +301,8 @@ class HooksWorkerClient
 
     @runner.hooks.afterAll (transactions, hookCallback) =>
 
-      # This is needed for transaction modification integration tests.
+      # This is needed for transaction modification integration tests:
+      # https://github.com/apiaryio/dredd-hooks-template/blob/master/features/execution_order.feature
```

Comment:
master? should probably tie this link to a specific tag or commit because it's a moving target. This could be a 404 in the future